notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1345_TEA_Vibration_audio_freq_analysis.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    sample_freq = fftpack.fftfreq(len(signal),time)
</pre><p>I think that you need to multiply sample_freq by the sample rate in order to get the frequency in Hertz.</p><p>If you do that you will see that the base frequency is around 3 kHz which correspond to the noise that can be heard in the audio file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>